### PR TITLE
SDK-254 [Python SDK] AWS KMS Key integration example

### PR DESCRIPTION
**Pull request by artemgolozubov**
Pull Request Checklist
======================

Please read this for our PR Culture:
------------------------------------
https://incountry.atlassian.net/wiki/spaces/ED/pages/46858869/Pull+requests+culture+Draft

Work relates to:
----------------
Link(s) to Jira Task/User Story/Bug/Epic:
- [SDK-254](https://incountry.atlassian.net/browse/SDK-254)

Code Quality (Before submitting PR)
------------
- [x] Ensure code compiles and passes Sonar Lint.
- [x] Do your best to ensure integration tests pass. If they don't then...
- [x] Remove, ignore, or mark pending any brittle tests.
- [x] Remove commented out code.

Target Release
--------------
- [x] Please put the target release vehicle this needs to be bundled with.
- [x] Did you increment the version? What is the new version?

### CODE REVIEW

**Review by igor-alekseev-dev — DISMISSED** [review]

File: examples/aws-kms.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+    new_record = {
+        "record_key": "recordKey-testAWSKMSKeys",
+        "key1": "key1",
+        "key2": "key2",
+        "key3": "key3",
+        "key10": "key10",
+        "profile_key": "profileKey",
+        "range_key1": 125,
+        "body": json.dumps({"test": "Test AWS KMS keys in Node.js SDK"}),
```
Comment (VladimirLevitskiy) [inline examples/aws-kms.py:64]: Please change the record body to '...Python SDK...'

**Review by VladimirLevitskiy — CHANGES_REQUESTED** [review]

**Review by VladimirLevitskiy — APPROVED** [review]
LGTM

File: examples/aws-kms.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        CiphertextBlob=ciphertext_blob,
+        KeyId=AWS_KMS_MASTER_KEY_ID_ARN,
+    )
+
+    if not response or not response["Plaintext"] or not isinstance(response["Plaintext"], bytes):
+        return
+
+    decrypted_key = response["Plaintext"]
+
+    decoded_key = base64.b64encode(decrypted_key).decode("utf8")
```
Comment (RommeroInCountry) [inline examples/aws-kms.py:42]: decrypted_key is already of bytes type. Python SDK allows bytes to be used as key straight away. No need to do all these transformations

**Review by RommeroInCountry — CHANGES_REQUESTED** [review]

**Review by RommeroInCountry — APPROVED** [review]

**Review by VladimirLevitskiy — APPROVED** [review]
LGTM

**Pull request merged**